website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-configuring-repositories.md

```diff
@@ -44,6 +44,14 @@ https://github.com/dbt-labs/jaffle_shop/compare/master..my-branch
 </TabItem>
 </Tabs>
 
+## Configuring Custom Branches on Repositories
+
+By default in Development Environments, dbt Cloud attempts to reference the `main` branch in connected repositories. If you want to use a different default branch name, you can configure dbt Cloud with a custom branch setting. 
```

Comment:
Is it still the case that the main branch must be named `main` or `master`?

Reply:
@amychen1776 dbt Cloud doesn't care what the base branch's name is.

Reply:
Ah epic, as long as it's set as the default? That's cool